docusaurus/android_versioned_docs/version-draft/01-basics/01-overview.mdx

```diff
@@ -0,0 +1,140 @@
+---
+slug: /
+---
+
+# Overview
+
+Built on top of the Stream Chat low-level client, the Stream Chat Android UI components enable you to easily build any type of chat or messaging experience for Android.
+
+We have UI component libraries available for both Android Views and Jetpack Compose. Each of these libraries provides an extensive collection of efficient and customizable UI components which enable you to quickly get started with little to no setup required. The libraries support:
+
+<!-- TODO validate list below. It's a copy from iOS, should we add/remove something? -->
```

Comment:
TODO

Reply:
Voice messages are not supported in Compose